Specs/Number/Arabic/NumberModel.json

```diff
@@ -187,47 +187,62 @@
     ]
   },
   {
-    "Input": "مائة وثلاثة وثلثين",
-    "NotSupportedByDesign": "javascript,python,java,dotnet",
```

Comment:
In the PR, test cases should only be marked as supported in .NET once the whole model passes. Otherwise tests will fail as they can't obtain the non-registered model.